driver/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
     mkdir $MAMBA_ROOT_PREFIX && \
     micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
         -c cpg -c bioconda -c conda-forge \
-        hail=$HAIL_VERSION analysis-runner bokeh selenium phantomjs \
+        hail=$HAIL_VERSION analysis-runner sample-metadata bokeh selenium phantomjs \
```

Comment:
Just double-checking whether we'd want to pin this version?

Reply:
Yeah good idea, I'll pin to start until there's more stability.